contracts/root/RootChainManager/RootChainManager.sol

```diff
@@ -303,19 +307,74 @@ contract RootChainManager is
             "RootChainManager: INVALID_USER"
         );
 
+        // Ether Predicate
+        // (Mintable)ERC20 Predicate
+        // (Mintable)ERC721 Predicate
+        // (Mintable)ERC1155 Predicate
+        if(tokenType == 0xa234e09165f88967a714e2a476288e4c6d88b4b69fe7c300a03190b858990bfc ||
```

Comment:
What if you add non-revertable `call('isVerifiable')` and call new method based on the result?

Reply:
Yes makes `if` condition look better, also less error-prone than typing those lengthy token types. Thanks ;)